src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Driver class that takes in input and performs certain functions based on input.
+ */
 public class Duke {
```

Comment:
this class is rather long, perhaps you could consider abstracting the methods out?